steal.lic

```diff
@@ -126,7 +126,7 @@ class Steal
     @bin_items.each do |item|
       if @slow_bin_speed
         bput("get #{item} from my #{@stealing_bag}", 'You get')
-        bput("put #{item} in bin", 'nods toward you as your .* falls into the azurite bin')
+        bput("put #{item} in bin", 'nods toward you as your .* falls into the azurite bin','nods toward you as your .* falls into the sturdy bin.')
```

Comment:
How about this instead, it may end up working in other towns also and is simpler.
```
bput("put #{item} in bin", 'nods toward you as your .* falls into the .* bin')